client/containers/domain-autocomplete/helpers/combine-domain-list.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 Uber Technologies Inc.
+//
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+const combineDomainList = ({ domainList, visitedDomainList }) => {
+  const domainListUuidList = domainList.map(domain => domain.domainInfo.uuid);
```

Comment:
how big do these lists get?  are they prior to the 15-limit or after?
if it's prior to that limit, probably worth switching this to be a map lookup - 100s might be visibly slow to do repeated lookups on a list like this.

Reply:
currently its before the 15 limit, but we can change the order to cap out to 15 earlier in the flow - within `DOMAIN_AUTOCOMPLETE_FILTERED_VISITED_DOMAIN_LIST` getter?

Reply:
Also right now there is no cap on visitedDomainList so this can grow infinitely. I will change the action in the next PR (#369) to cap out to a max of 15.